assets/javascript/progressive-reveal.js

```diff
@@ -12,11 +12,8 @@ function inputClicked(e, target) {
     target = target || helpers.target(e);
     var shown;
     _.each(groups[target.name], function (input) {
-        var id = input.getAttribute(toggleAttr);
-        // check if the element supplied is part of an {id}-panel
-        // if so then toggle this parent element to also toggle
-        // associated labels and legends
-        var toggle = document.getElementById(id + '-panel') || document.getElementById(id);
+        var id = input.getAttribute('aria-controls');
```

Comment:
There isn't a test checking that the `aria-controls` attribute is added. If we're relying on this elsewhere we should add a test.